IDEAS/Buildings/Components/Shading/Screen.mo

```diff
@@ -1,13 +1,26 @@
 within IDEAS.Buildings.Components.Shading;
 model Screen "Controllable exterior screen"
-  extends IDEAS.Buildings.Components.Shading.Interfaces.PartialShading(
-                                                             final controlled=true);
+  extends IDEAS.Buildings.Components.Shading.Interfaces.PartialShadingDevice(
+    TSha = TShaScreen,
+    TDryBul_internal = limiter.y*TSha + (1-limiter.y)*Te_internal,
+    epsSw_shading = 1 - shaCorr,
+    final controlled = true,
+    TEnvExpr(y = TEnv_screen),
+    TeExpr(y = TDryBul_internal));
```

Comment:
Are the new parameter declarations of the extension deliberately not set to final?